cernopendata/modules/fixtures/cli.py

```diff
@@ -190,10 +190,10 @@ def records(skip_files, files, profile, mode):
         record_json = glob.glob(os.path.join(data, '*.json'))
 
     for filename in record_json:
-        name = filename.split('/')[-1]
-        if name.startswith('opera'):
-            click.echo('Skipping opera records ...')
-            continue
+        # name = filename.split('/')[-1]
```

Comment:
Since we are merging to `qa`, please leave the OPERA skipping in place...

Reply:
Sorry, I picked up unnecessary commit(s) from master. Fixed.